example-server/app/public/form.js

```diff
@@ -558,8 +571,80 @@ app.controller('AutoSearchController', [
     scope.eventString = JSON.stringify(scope.event, null, 2);
     scope.error       = '';
 
+    scope.overrideApiKey = function () {
+      scope.allowOverrideApiKey = !scope.allowOverrideApiKey;
+    };
+
+    scope.overrideSearch = function () {
+      scope.allowSearchOverride = !scope.allowSearchOverride;
+      if (!scope.allowSearchOverride) {
+        scope.searchPath = '';
+      }
+    };
+
+    scope.overrideBeacon = function () {
+      scope.allowBeaconOverride = !scope.allowBeaconOverride;
+      if (!scope.allowBeaconOverride) {
+        scope.beaconPath   = '';
+        scope.manualBeacon = false;
+      }
+    };
+
+    scope.beaconPathChange = function () {
+      console.log('changed');
+      scope.manualBeacon = scope.beaconPath.length !== 0;
+    };
+
     scope.send = function () {
-      sendEvent(scope, sentTimeout, tracker, 'sendAutoSearchEvent');
+      try {
+        scope.jsonError        = false;
+        const searchBodyObject = JSON.parse(scope.searchBody);
+
+        let searchUrl;
+        if (scope.searchPath.length > 0) {
+          searchUrl = scope.searchPath;
+        } else {
+          searchUrl = "https://" + scope.customerId + "-cors.groupbycloud.com/api/v1/search";
+        }
+
+        let beaconUrl;
+        if (scope.beaconPath.length > 0) {
+          beaconUrl = scope.beaconPath;
+        } else {
+          beaconUrl = "https://" + scope.customerId + "-cors.groupbycloud.com/wisdom/v2/internal/beacon"
+        }
+
+        // http.defaults.headers.post['Content-Type'] = 'text/plain';
+        http.post(searchUrl, searchBodyObject).then((response) => {
+          console.log(response);
+          scope.event.search.id = response.data.id;
+          scope.eventString     = JSON.stringify(scope.event, null, 2);
```

Comment:
This is intended to display a stringified version of the event in the UI. When `scope.eventString` is populated, it's displayed for some period before being wiped and disappearing again. So the use in this case is ok from the context of data duplication. 

However, in this specific case, this code will have no effect. `scope.eventString` is being almost immediately overwritten by the autoSearch event, so this output will never be visible. Also, the full body of this request is already visible in a field, so there is no need to make it visible here.

So we can remove 621.